src/components/frame/Extensions/pages/LazyConsensusPage/LazyConsensusPage.tsx

```diff
@@ -54,65 +68,55 @@ const LazyConsensusPage: FC = () => {
-                  appearance={{ size: 'large' }}
-                />
-                {extensionData?.name?.defaultMessage && (
+            <div className="flex justify-between flex-col flex-wrap sm:items-center sm:flex-row sm:gap-6">
+              <div className="flex flex-col sm:items-center sm:flex-row sm:gap-2 sm:grow">
+                <div className="flex items-center shrink-0">
+                  <Icon
+                    name={extensionData.icon}
+                    appearance={{ size: 'large' }}
+                  />
                   <h4 className="ml-2 text-xl font-semibold text-gray-900">
-                    {extensionData?.name?.defaultMessage as React.ReactNode}
+                    {formatMessage(extensionData.name)}
                   </h4>
-                )}
-
-                {/* @TODO: add condition to show/hide pills */}
-                <div className="flex justify-between items-center w-full sm:w-auto mt-5 sm:mt-0">
+                </div>
+                <div className="flex items-center justify-between gap-4 mt-4 sm:mt-0 sm:grow">
                   <ExtensionStatusBadge
                     mode="governance"
-                    text={formatMessage({ id: 'extensionsPage.governance' })}
+                    text={formatMessage({ id: 'status.governance' })}
                   />
-                  <span className="flex sm:hidden text-gray-400 text-sm">{`17,876 ${formatMessage(
-                    {
-                      id: 'active.installs',
-                    },
-                  )}`}</span>
-                </div>
-              </div>
-
-              <div className="sm:ml-4">
-                <div className="flex gap-6 items-center justify-end min-h-[2rem]">
-                  <span className="hidden sm:flex text-gray-400 text-sm">{`17,876 ${formatMessage(
-                    {
-                      id: 'active.installs',
-                    },
-                  )}`}</span>
-
-                  {isEnableButtonVisible && (
-                    <Button
-                      mode="primarySolid"
-                      type="submit"
-                      isFullSize={isMobile}
-                    >
-                      <p className="text-sm font-medium">
-                        {formatMessage({ id: 'extension.enableButton' })}
-                      </p>
-                    </Button>
+                  {activeInstalls >= ACTIVE_INSTALLED_LIMIT ? (
+                    <p className="text-gray-400 text-sm">
+                      {activeInstalls.toLocaleString('en-US')}{' '}
+                      {formatMessage({ id: 'active.installs' })}
+                    </p>
+                  ) : (
+                    <ExtensionStatusBadge
+                      mode="new"
+                      text={{ id: 'status.new' }}
+                    />
                   )}
-                  <ActionButtons extensionData={extensionData} />
                 </div>
+                {isEnableButtonVisible && (
```

Comment:
are you sure that this button is necessary?

Reply:
Removed